tests/test_integration.py

```diff
@@ -518,7 +518,7 @@ def test_igvm_migration_log(self, performance_value, mock_time):
 
         src_hv = self.vm.hypervisor.dataset_obj['hostname']
         cpu_usage_vm_src = self.vm.hypervisor.estimate_vm_cpu_usage(self.vm)
-        timestamp = 1234567890
+        timestamp = 1234567890000
```

Comment:
Shouldn't this test timestamp be updated to the new format too, in case we ever restrict in Serveradmin?